crates/cargo-wdk/src/cli.rs

```diff
@@ -66,20 +68,103 @@ pub enum Subcmd {
     Build(PackageProjectArgs),
 }
 
-// TODO: Implement workspaces
+#[derive(Debug, Clone)]
+pub enum DriverType {
+    KMDF,
+    UMDF,
+    WDM,
```

Comment:
Rust standard casing for enum variants is pascal case so better to rename these to `Umdf`, `Kmdf and `Wdm`